index.html

```diff
@@ -0,0 +1,439 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Ghost host</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="font/fontawesome-free-5.4.1-web/css/all.css">
+    <link rel="shortcut icon" href="images/icon.png" type="image/png">
+</head>
+
+<body>
+
+    <header>
+        <nav class="menu">
+            <h1>Gost Host</h1>
+            <div class="logo"></div>
+            <div><a href="#">Home</a></div>
+            <div><a href="#domain">Domain</a></div>
+            <div><a href="#hosting">Hosting</a></div>
+            <div><a href="#">Clients</a></div>
+            <div><a href="#">Blog</a></div>
+            <div><a href="#support">Support</a></div>
+        </nav>
+
+        <div class="greeting">
+            <div></div>
```

Comment:
DO YOU REALLY NEED DIV TO PLACE CONTENT LOWER ? MAYBE MARGIN OR PADDING CAN HELP YOU ?